Gemfile.lock

```diff
@@ -7,7 +7,7 @@ GIT
 
 GIT
   remote: https://github.com/ideacrew/aca_entities.git
-  revision: 63c341af12eb303f38e6ce944209d67294bc0dd4
+  revision: 87b6170325d9040bc8a327481dda5cfcc8501514
```

Comment:
@polographer this most definitely could be a breaking change and cannot be considered pure refactoring

Reply:
removed